pages/map.js

```diff
@@ -1,25 +1,50 @@
 import React from 'react';
 import Head from 'next/head';
 import styled from 'styled-components';
+import Layout from '../components/Layout';
 
 class Map extends React.Component {
   render() {
+    const pageTitle = 'Maps of TJI Data';
     return (
       <React.Fragment>
         <Head>
           <title>Texas Justice Initiative | Map</title>
         </Head>
-        <MapIframe src="https://map-viewer.texasjusticeinitiative.org/" />
+        <Layout fullWidth>
+          <Main>
+            <h1>{pageTitle}</h1>
+            <p>
+              This page contains map view of TJI datasets, including Custodial Deaths and Officer Involved Shootings in
+              Texas.
```

Comment:
What's the context that was missing from this page that we want to make more clear? I think most users could figure this out from existing context on the page. Is there something else missing here that we want to get across?

Reply:
Yeah, I felt like we may need something here, but after adding a title and description area, I'm not convinced we do... Or what we should say.